Nashua.txt

```diff
@@ -0,0 +1,8 @@
+## Nashua, NH
+
```

Comment:
Being at Portland for 6 months 10+ years ago. Like the fist few months when it was sunny and warm. But then rainy season started and no sun for weeks:(. Good city for all kinds of outdoor activities, though.

Thanks for reviewing:)